Makefile

```diff
@@ -1,12 +1,12 @@
 GIR = gir/target/release/gir
 CONFIGS = $(wildcard conf/gir-*.toml)
-LIBS = $(CONFIGS:conf/gir-%.toml=%-sys/src/lib.rs)
+LIBS = $(CONFIGS:conf/gir-%.toml=%-sys/Cargo.toml)
 
 libs : $(LIBS)
 
-%-sys/src/lib.rs : conf/gir-%.toml $(GIR)
+%-sys/Cargo.toml : conf/gir-%.toml $(GIR)
```

Comment:
This replacement has a potential to produce surprising results: failure to regenerate if you edit both `foo-sys/Cargo.toml` and `conf/gir-foo.toml`.